source/layouts/layout.slim

```diff
@@ -12,6 +12,7 @@ html
     - categories.each do |category|
       = feed_tag :atom, "#{config[:platform_url]}/categories/#{category.slug}.xml", title: "#{config[:platform_title]} – #{category.title} articles"
     = stylesheet_link_tag :blog
+    script src="//platform-api.sharethis.com/js/sharethis.js#property=58ea34c2b23bb10011d6ce9a&product=inline-share-buttons" type="text/javascript"
```

Comment:
@skatkov delayed loading request is complete... As long as this script is the only bit of JS logic on the project I've deiced not to involve heavy third-party libraries (such as jQuery or domready.js) to detect page loading state and scripts requirement and just use a [piece of vanilla JS](https://github.com/saberespoder/sepcontent-v2/pull/21/files#diff-80ba1f9592234480fc349d018d11eb4cR1) for this purpose.